cuda_pathfinder/cuda/pathfinder/_utils/conda_env.py

```diff
@@ -0,0 +1,41 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+import functools
+import os
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Literal, Optional
+
+# https://docs.conda.io/projects/conda-build/en/stable/user-guide/environment-variables.html
+
+BUILD_STATES = ("RENDER", "BUILD", "TEST")
```

Comment:
During re-rendering, there is no actual conda env created nor packages installed, it can be removed
```suggestion
BUILD_STATES = ("BUILD", "TEST")
```